### ISSUE TITLE
[Build] Not LESS 4 compatible  

### ISSUE THREAD

**Issue by arichiardi** [fomantic/Fomantic-UI#1818]
# Bug Report
This is a problem I detected when upgrading Less to version 4 and therefor this belongs to https://github.com/fomantic/Fomantic-UI-LESS

I have discovered that Less 4 adds new math-related options (https://github.com/less/less.js/pull/3274) and enforces parenthesis for division (https://github.com/less/less.js/pull/3573/commits/58c89eecf9fef17248dacab1d4925c236c7ac17e).

This breaks compilation, see below.

## Steps to reproduce
1. Bump Less to version 4
2. Compile Fomantic-UI-LESS

## Expected result
Successful compilation.

## Actual result

The output is:

```
[lessc] - ArgumentError: Error evaluating function `unit`: the first argument to unit must be a number. Have you forgotten parenthesis? in /home/kapitan/.boot/cache/tmp/home/kapitan/git/ar-website/8lt/cz07er/css/themes/default/globals/site.variables on line 653, column 20:
652 @largeRaw        : unit( round(@largeSize * @emSize) / @emSize );
653 @bigRaw          : unit( round(@bigSize * @emSize) / @emSize );
654 @hugeRaw         : unit( round(@hugeSize * @emSize) / @emSize );
```
## Version
2.8.7

**Comment by lubber-de** [comment]
Thanks for the bug report. It's fixed by #1819.

**Comment by arichiardi** [comment]
Well thank you for the patch!
